src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -529,7 +533,13 @@ def get_op_handler(operation):
     import types
 
     for rt in ResourceType:
-        if operation.startswith(rt.import_prefix):
+        if operation.startswith(rt.import_prefix+".operations."):
```

Comment:
@derekbekoe I wanted to point that, that I realized working on it this afternoon. if you cal `cli_command` with an operation like `azure.mgmt.compute.operations.availability_sets_operations#create_or_update`, now I correctly extract `availability_sets` from the string to get the exact api version involved.